src/site/paragraphs/linkTeaser.drupal.liquid

```diff
@@ -33,21 +33,16 @@
   data-links-list-header="{{ link.title | escape }}"
   data-links-list-section-header="{{ section_header | escape }}"
   {% if parentFieldName === 'field_spokes' %}class="hub-page-link-list__item"{% endif %}>
-    {% if link.title != empty %}
-      <{{ header }} class="{{ headerClass }}">
-        <a href="{{link.url.path}}" class="vads-u-text-decoration--underline"
-          {% if linkTeaser.options["target"] %}
-            target="{{ linkTeaser.options["target"] }}"
-            rel="noopener noreferrer"
-          {% endif %}
-        >
-          {{ link.title }}
-        </a>
-      </{{ header }}>
-      {% if parentFieldName === "field_spokes" %}
-          <img class="all-link-arrow" src="/img/arrow-right-blue.svg" alt="Blue arrow pointing right">
-      {% endif %}
-    {% endif %}
+    <a href="{{link.url.path}}"
+      {% if linkTeaser.options["target"] %}target="{{ linkTeaser.options["target"] }}"{% endif %}
+      >
+        {% if link.title != empty %}
+            <{{ header }} class="{{ headerClass }}">{{ link.title }}</{{ header }}>
+            {% if parentFieldName === "field_spokes" %}
+                <img class="all-link-arrow" src="/img/arrow-right-blue.svg" alt="right-arrow">
```

Comment:
I know this is a revert, but can we keep the `alt` text "Blue arrow pointing right"?

Reply:
We’re going to do this fix again so we’ll get it on the next go-around. I don’t want to wait another 45 for CI to finish.